internal/learn/future/shimmer.md

```diff
@@ -0,0 +1,33 @@
+---
+id: shimmer
+title: Shimmer
+description: Introduction to the emerging incentivized staging network
+keywords:
+  - Incentivized
+  - staking
+  - SMR
+---
+
+IOTA Foundation [introduced](https://blog.iota.org/introducing-iota-staking/) Shimmer. An incentivized staging network.
+It can be used to work with upcoming IOTA updates before they arrive on the mainnet. It will also have its own token called [SMR](https://shimmer.network/token).
+To fairly distribute tokens on the new network, you can stake your IOTA tokens before launch.
+
+## Pre-launch staking
+
+Before the Shimmer network goes online, the IOTA Foundation will announce a start date for "pre-launch" staking.
+You will be able to make a transaction to yourself with a certain payload to declare that you want to stake that amount of IOTA tokens to receive SMR. Staking will now be possible for 90 days. You will earn 1 SMR for every 1Mi every 10 seconds.
+Let's take a closer look at how this works.
+
+### Technical overview
+
+Those who are already familiar with the Build/Burn Vote process will recognize the similarities between the Vote and Staking, as it is performed by the same Hornet plugin.  
+Hornet operators can enable this plugin. Hornet will then (after a certain milestone) start accumulating all outputs that have the specific Staking payload every 10 seconds. Until the 90 days are up.  
+Not clear enough? Let's use an example:  
```

Comment:
```suggestion
Let's see an example of how this would work:  
```

Could be perceived a bit passive aggressive

Reply:
Sounds good